templates/core/.env.sample

```diff
@@ -11,3 +11,7 @@ AAD_TENANT_ID=__CHANGE_ME__
 API_CLIENT_ID=__CHANGE_ME__
 API_CLIENT_SECRET=__CHANGE_ME__
 SWAGGER_UI_CLIENT_ID=__CHANGE_ME__
+
+# Automation accounts (e.g. bundle registration)
+AUTOMATION_ADMIN_ACCOUNT_CLIENT_ID=__CHANGE_ME__
+AUTOMATION_ADMIN_ACCOUNT_CLIENT_SECRET=__CHANGE_ME__
```

Comment:
can you add a new line at the end of file please?

Reply:
done 😄